schema/pet-insurance.json

```diff
@@ -152,17 +201,649 @@
                 "petType": {
                     "type": "string",
                     "description": "",
-                    "example": "dog"
+                    "example": "dog",
+                    "enum": [
+                        "dog",
+                        "cat"
+                    ]
                 },
                 "petGender": {
                     "type": "string",
                     "description": "",
                     "example": "Male"
                 },
+                "breedType": {

```

Comment:
What is this and how is it different from `breedName`? The example, `"Male"`, makes me think it's `gender`, not `breedType`. A short description or better example would help.

Reply:
Isn't it under `petGender`? I don't understand. I will add description.

Reply:
`breedType`'s example is `"Male"`. Sounds a lot like gender, not type. Is this example correct?

Reply:
In that case, it isn't.